.gitignore

```diff
@@ -32,3 +32,7 @@
 /config/secrets/*
 
 .env.development
+.env.test
+
+# Ignore credentials
+/config/credentials.yml.enc
```

Comment:
I think this is causing your tests to fail on Travis. Why did you add it?

Reply:
Somehow. I need to remove the credentials that I have locally and generate my own. When I do this, this file is being changed so I thought of ignoring it.